### DISCUSSION TITLE
Attributes dynamic to certain condition (e.g. window_gap based on no of windows/frames)?

### DISCUSSION THREAD

**Discussion by Animeshz** [herbstluftwm/herbstluftwm#1456] [Q&A] [answered]
My point of interest: Is there a way to set window_gap to 0 if current tag has a single frame and single window inside it?

**Comment by t-wissmann** [comment, marked as answer]
There is only `smart_frame_surroundings` and `smart_window_surroundings`. but the frame-setting only takes the number frames into account and the window-setting only takes the number of windows in a frame into account.

**Reply by Animeshz** [reply]
Just one more question, can I write a rule to do this (`smart_window_surroundings`) for a few applications (e.g. browser and music player), because their scrolling is better when attached without gaps.

**Reply by t-wissmann** [reply]
Unfortunately not. the setting is global for all applications.
